src/runtime/config/configuration-clh.toml.in

```diff
@@ -449,6 +449,11 @@ experimental=@DEFAULTEXPFEATURES@
 # (default: false)
 # enable_pprof = true
 
+# CreateContainer request timeout in seconds.
+# If specified, indicates the CreateContainer request timeout needed for the workload(s)
+# If unspecified then it will be set @DEFCREATECONTAINERREQUESTTIMEOUT@ second(s) 
```

Comment:
I wonder if it would be clearer to day something like:
```suggestion
# Indicates the CreateContainer request timeout needed for the workload(s)
# Note it using guest_pull this includes the time to pull the image inside the guest
# Defaults to @DEFCREATECONTAINERREQUESTTIMEOUT@ second(s) 
```

Reply:
Done. Thanks @stevenhorsman